core/backend/package.json

```diff
@@ -67,7 +67,7 @@
     "@types/ws": "^6.0.4",
     "chai": "^4.1.2",
     "chai-as-promised": "^7",
-    "cpx": "^1.5.0",
+    "cpx2": "^3.0.0",
```

Comment:
Why are we updating to 3.x if 4.0 is the latest?

Reply:
See this comment, https://github.com/iTwin/itwinjs-core/pull/2714#issuecomment-970462569.

It's due to use still supporting Node 12.